[2022-05-16 16:20:02,865] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: UTE_THONG_BAO_V1.t2 2022-05-16T16:15:00+00:00 [queued]>
[2022-05-16 16:20:02,875] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: UTE_THONG_BAO_V1.t2 2022-05-16T16:15:00+00:00 [queued]>
[2022-05-16 16:20:02,875] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-05-16 16:20:02,875] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2022-05-16 16:20:02,875] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-05-16 16:20:02,883] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): t2> on 2022-05-16T16:15:00+00:00
[2022-05-16 16:20:02,886] {standard_task_runner.py:52} INFO - Started process 11397 to run task
[2022-05-16 16:20:02,889] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'UTE_THONG_BAO_V1', 't2', '2022-05-16T16:15:00+00:00', '--job-id', '83', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test/test.py', '--cfg-path', '/tmp/tmp5kjehots', '--error-file', '/tmp/tmp1tbar6p5']
[2022-05-16 16:20:02,889] {standard_task_runner.py:77} INFO - Job 83: Subtask t2
[2022-05-16 16:20:02,921] {logging_mixin.py:104} INFO - Running <TaskInstance: UTE_THONG_BAO_V1.t2 2022-05-16T16:15:00+00:00 [running]> on host 65d21844f899
[2022-05-16 16:20:02,955] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=UTE_THONG_BAO_V1
AIRFLOW_CTX_DAG_ID=UTE_THONG_BAO_V1
AIRFLOW_CTX_TASK_ID=t2
AIRFLOW_CTX_EXECUTION_DATE=2022-05-16T16:15:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-16T16:15:00+00:00
[2022-05-16 16:20:02,956] {logging_mixin.py:104} INFO - [!] DATA CLEAR !!
[2022-05-16 16:20:02,956] {python.py:151} INFO - Done. Returned value was: None
[2022-05-16 16:20:02,960] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=UTE_THONG_BAO_V1, task_id=t2, execution_date=20220516T161500, start_date=20220516T162002, end_date=20220516T162002
[2022-05-16 16:20:02,981] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-05-16 16:20:03,020] {local_task_job.py:151} INFO - Task exited with return code 0
